createdisk-library.sh

```diff
@@ -172,7 +172,7 @@ function copy_additional_files {
 
 function prepare_hyperV() {
 
-    if [ "${ARCH}" != "ppc64le" ]; then
+    if [ "${ARCH}" == "ppc64le" ]; then
```

Comment:
shouldn't this be part of first commit where this logic is added?

Reply:
yea i could squash this if needed - i messed this up